.ci/jobs/beats-webhook.yml

```diff
@@ -0,0 +1,21 @@
+---
+- job:
+    name: apm-shared/beats-webhook
+    display-name: Beats webhook
+    description: List for webhook events
+    concurrent: true
+    view: APM-CI
+    project-type: pipeline
+    pipeline-scm:
+      script-path: .ci/beats-webhook.groovy
+      scm:
+        - git:
+            url: git@github.com:elastic/apm-pipeline-library.git
+            refspec: +refs/heads/*:refs/remotes/origin/* +refs/pull/*/head:refs/remotes/origin/pr/*
+            wipe-workspace: 'True'
```

Comment:
Nit: It's a bit odd to see this value quoted. Is there a reason for that?

Reply:
Indeed, that's a C&P from https://github.com/elastic/apm-pipeline-library/blob/master/.ci/jobs/apm-schedule-daily.yml#L19

It should be safe to use a bool as far as I read in https://docs.openstack.org/infra/jenkins-job-builder/scm.html?highlight=wipe%20workspace#scm.git

Let me do it now

Reply:
```suggestion
            wipe-workspace: true
```